mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -129,79 +127,75 @@ var PerfCmd = &cobra.Command{
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		// Importing SMPS Configuration from the file
+		var req *http.Request
 		if filePath != "" {
-			var t models.PerformanceSpec
-			err := yaml.Unmarshal([]byte(filePath), &t)
+			smpsConfig, err := ioutil.ReadFile(filePath)
+
 			if err != nil {
-				return errors.Wrapf(err, utils.PerfError(fmt.Sprintf("failed to unmarshal yaml file %s", filePath)))
-			}
-			if testDuration == "" {
-				testDuration = fmt.Sprintf("%fs", t.EndTime.Sub(t.StartTime).Seconds())
+				return err
 			}
-			if testURL == "" {
-				testURL = t.EndpointURL
-			}
-			if concurrentRequests == "" {
-				concurrentRequests = fmt.Sprintf("%d", t.Client.Connections)
+
+			req, err = http.NewRequest("POST", mctlCfg.GetPerf().LoadTestSmpsURI, bytes.NewBuffer(smpsConfig))
```

Comment:
Just a doubt, what is the behaviour when I have given an SMPS file as well as I have entered some arguments while executing the command?

Reply:
Currently, the other configs are ignored. I am not sure if it should be that way though. Currently there is an API  endpoint `/api/load-test-smps` that we are using that expects and SMPS file in the request. So if we want to override the test details in the SMPS then we will have to modify the SMPS file and then send it. Also, considering we are moving towards multiple clients and multiple endpoints, it might become tedious to specify everything in detail using CLI flags. We should discuss and decide on this. @leecalcote ?